lib/links/parser.rb

```diff
@@ -2,51 +2,58 @@
 
 class Parser
 
-    def initialize(url)
-        @elements = {a: ["href"],
-                img: ["src"],
-                script: ["src"],
-                frame: ["src"],
-                iframe: ["src"],
-                link: ["href"],
-                source: ["src", "srcset"],
-                track: ["src"] }
-
-        @url = url
+  def initialize(url)
+    @elements = { a: ['href'],
```

Comment:
This could be a class variable or class instance variable, couldn't it?

[Here](http://maximomussini.com/posts/ruby-class-variables/)'s a resource to read on that.